client/components/navbar.jsx

```diff
@@ -1,30 +1,115 @@
 import React from 'react';
+import styled, { css } from 'styled-components';
 import logo from '../igniter.png';
-import search from '../search.png';
 
 const NavBar = () => (
-  <div className="">
-    <button type="submit" className="navbarOption mr">
-      Explore
-    </button>
+  <div>
+    <Table>
+      <tr>
+        <TableDimension navItemRight>
+          <Button navbar>
+            Explore
+          </Button>
+        </TableDimension>
+        <TableDimension navItemRight>
+          <Button navbar>
+            Start a project
+          </Button>
+        </TableDimension>
+        <TableDimension logo>
+          <Image src="https://d3mlfyygrfdi2i.cloudfront.net/kickstarter-logo-color.png" alt="logo" logo />
+        </TableDimension>
+        <TableDimension navItemLeft>
+          <Button navbar>
+            Search
+          </Button>
+        </TableDimension>
+        <TableDimension navItemLeft>
+          <Button navbar>
+            Sign In
+          </Button>
+        </TableDimension>
+      </tr>
+    </Table>
+    <Horizontal />
+  </div>
+);
 
-    <a href="google.com" className="navbarOption mr">
-      Start a project
-    </a>
+const Table = styled.table`
+  width: 100%;
+`;
 
-    <img src={logo} alt="igniter" />
+const TableDimension = styled.td`
+  ${props => props.navItemRight && css`
+    width: 10%;
+    text-align: left;
+  `}
+  ${props => props.navItemLeft && css`
+    width: 10%;
+    text-align: right;
+  `}
+  ${props => props.logo && css`
+    width: 50%;
+    text-align: center;
+  `}
+`;
 
-    <button type="submit" className="navbarOption">
-      Search
-      <img src={search} alt="search" className="searchIcon"/>
-    </button>
+const Button = styled.button`
+  ${props => props.navbar && css`
+    border: none;
+    outline: none;
+    color: 2b2727;
+    font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+    font-size: 1em;
+    &:hover {
+      color: #034752;
+    }
+  `}
+`;
 
-    <button type="submit" className="navbarOption">
-      Sign In
-    </button>
+const Image = styled.img`
+  ${props => props.logo && css`
+    width: 20%;
+    height: 20%;
+  `}
+`;
+
+const Horizontal = styled.hr`
+  border-color: #034752;
+`;
 
-    <hr />
-  </div>
-);
 
 module.exports = NavBar;
+
```

Comment:
It seems that you have the same code above.

Reply:
@despinoz I made some changes that I thought would be better closer to the actual navbar on kickstarter.